### ISSUE TITLE
`dep run` makes unhandled promise rejection

### ISSUE THREAD

**Issue by watilde** [watilde/dep#17]
### Summary
> unhandled promise rejection looks to be a bug. https://github.com/watilde/dep/issues/7

### Steps to reproduce behavior
?

### Expected behavior
It should work without errors.

### Actual behavior
```console
$ dep run pretest
(node:25609) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): /bin/sh: 1: eslint: Permission denied
(node:25609) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by watilde** [comment]
I believe this was fixed by https://github.com/watilde/dep/pull/21. The test is here: https://github.com/watilde/dep/blob/master/test/30-run.js.

**Comment by watilde** [comment]
The patch was released as `v0.14.4`. Thanks :)
